.travis.yml

```diff
@@ -29,17 +24,18 @@ before_install:
       make && make install;
       cd ..;
     fi
+
+install:
+  - nvm install $TARGET_NODE_VERSION
+  - nvm use $TARGET_NODE_VERSION
   - node --version
   - npm --version
   - npm install -g elm@$ELM_VERSION
   - mv $(npm config get prefix)/bin/elm $(npm config get prefix)/bin/elm-old
-  - printf '%s\n\n' '#!/bin/bash' 'echo "Running elm make with sysconfcpus -n 2"' '$TRAVIS_BUILD_DIR/sysconfcpus/bin/sysconfcpus -n 2 elm-old "$@"' > $(npm config get prefix)/bin/elm
+  - printf '%s\n\n' '#!/bin/bash' 'echo "Running elm-make with sysconfcpus -n 2"' '$TRAVIS_BUILD_DIR/sysconfcpus/bin/sysconfcpus -n 2 elm-old "$@"' > $(npm config get prefix)/bin/elm
   - chmod +x $(npm config get prefix)/bin/elm
-
-install:
-  - elm package install --yes
   - npm install
+  - pushd examples && elm make Main.elm && popd
```

Comment:
I think compilation should not be part of install

Reply:
Moved to `script`